### ISSUE TITLE
Feedback week 4

### ISSUE THREAD

**Issue by lluisgomez** [acasadevall/VR-Team4#2]
I've been taking a look to the Overleaf article sections and Google slides related with the work done during weeks 3/4 and I have some comments about how to improve them. 

First of all I think the state of the art review can be improved by adding a bit more of context about the object detection problem and other methods that just YOLO and SSD. 

You have a lot of plots in the article that in my opinion can be placed with a much better layout. e.g. use a double column figure and three plots per row. Also, try to summarize better the "message" of those plots in the text, maybe in a single paragraph for all them. I think is difficult for the reader to focus on each plot in the way you are presenting them on section 3.2.2. 

Please use the correct names for the metrics: "recall" -> "Avg. recall", "IoU" -> "Avg. IoU".

Regarding the results shown on section 4.2, tables 2 and 3 are not what I expected to see. What we want is to compare the methods by their final F-score on the test set. The other metrics (avg. recall and avg. IoU) are only "helper" metrics that we use during training to understand if the model is learning well.
Also, loss values between SSD and YOLO are not comparable (they use a different loss function) and thus they are not meaningful in a method comparison table.
In any case, all this section needs a much more of critical appreciation of the results. Which was the best result, why? conclusions on tables 2 and 3? etc.

I could not find any detail/analysis about the datasets used in your experiments. This may also help you to explain the obtained results. Images with qualitative results may also be helpful if you have space and consider them relevant.

Regarding the presentation slides for these two weeks, a few comments: 
           
- On slide 14 you say YOLO was trained from scratch but it was not (in the experiment we provided).
- Also on slide 14: "without metrics" is not a relevant info. You train a model with a loss function, and by convenience you can use other metrics but they are not indispensable.
- On slide 15: we want to compare methods by their F-Score. Accuracy does not mean anything on object detection task. For what I can see what you are showing on slide 15 is the avg. recall. right?
- You should consider adding a slide with qualitative results. It may help to explain where the networks work well, where they fail, etc.

Finally, I see in your code that you have the SSD integration almost done, but the provided results are not conclusive. You only show loss values on table 4 of your article, but the provided values are too high for the SSD loss. I've downloaded the weights of your SSD model trained on Udacity and it seems there is some problem on your integration. Still, I appreciate the work you have done to integrate this network, I know it was not an easy task.

All in all I guess you are going to obtain a good mark for these two weeks' assignment, although not the maximum mark as there is no proposal to overcome the challenge of the train/test biases on the Udacity dataset, and the SSD integration is not fully complete.

Congratulations for all the work done!
